docs/ref/forms/fields.txt

```diff
@@ -1357,6 +1357,15 @@ generating choices. See :ref:`iterating-relationship-choices` for details.
 
 .. _iterating-relationship-choices:
 
+.. method:: ModelMultipleChoiceField.validate_choices(queryset, field_name, selected_choices)
```

Comment:
This should be above the ` .. _iterating-relationship-choices: `. Also `versionadded:: 4.0` annotation and release notes are missing, see https://github.com/django/django/pull/7181#discussion_r131785303.

Reply:
I was not sure what the new use-case would be regarding the super() call, I would gladly accept some help with this.

Reply:
We should have a valid use case if we want to add a new hook :thinking: It can be used to avoid a queryset evaluation, but I'm not sure it's enough to justify adding it. It's really niche and was more of a workaround than a real need (see ticket-27148). I'm going to say `wontfix` unless there's a more legitimate use-case.

Reply:
Yeah I understand, I just wanted to fix the issues in the PR and code some and if it is needed merge it. But I am fine with a wontfix that closes the ticket as well.